README.md

```diff
@@ -18,6 +18,24 @@ If pytest is not available, you can also run tests from the package root folder
 python3 -m unittest discover
 ```
 
+## Quickstarter
+To use GeoLlama, first create a YAML configuration file by running the command
+```
+geollama generate-config [--output-path PATH]
+```
+The configuration will be generated in the current working directory with default parameters if the `--output-path` flag is not provided.
```

Comment:
slightly confusing wording - if i specify the output path will the default parameters still be in the yaml?

Reply:
Oh, I meant 

> A YAML configuration file will be generated with default parameters. If the `--output-path` flag is not provided, the file will be generated at `./config.yaml` by default.

Will be amended.